model/security/update_all_distinguished_names/operations.smithy

```diff
@@ -0,0 +1,27 @@
+
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+use opensearch.openapi#vendorExtensions
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/latest/security/access-control/api/#update-all-distinguished-names"
+)
+
+@vendorExtensions(
+    "x-operation-group": "security.update_all_distinguished_names",
+    "x-version-added": "1.0",
+)
+@idempotent
+@suppress(["HttpUriConflict"])
+@http(method: "PATCH", uri: "/_plugins/_security/api/nodesdn")
+@documentation("Bulk update of distinguished names.")
+operation UpdateAllDistinguishedNames {
```

Comment:
I feel like for consistency this should be named `PatchDistinguishedNames` as with all other security plugin resources and configuration